mango/table/pytups_table.py

```diff
@@ -86,18 +95,31 @@ def unique2(self) -> "Table":
             )
 
     def sorted(self, **kwargs) -> "Table":
-        raise NotImplementedError(
-            "A list of dict cannot be sorted. Use order_by instead"
-        )
+        try:
+            return Table(super().sorted(**kwargs))
+        except:
```

Comment:
Can this be hanlded with if-else statements instead of catching all exceptions and raising a "NotImplementedError". I guess it works but doesn´t seem like a good practice to do

Reply:
you have a good point here but I am not sure it is the best thing to do in this situation.

Table class is a list of dict and a subclass of Tuplist which is a list of tuples or dict and a subclass of a normal list.

Using sorted on a list of dict doesn't work but if you have a list of tuples or a normal list in your Table, it might work.
The idea of having the try-except is "if for some reason it works with your data, we allow it, otherwise we give you an error message which is more clear than the normal list error message".

The alternatives are:

- always raise an error.
- never raise an error an let the normal list error handle it.
- use sorted as an allias for order_by.
- check the type of the first object of the list and raise error if it is a dict.

I think I can at least narrow down the error to catch which is TypeError